test/components/app-test.js

```diff
@@ -0,0 +1,19 @@
+const assert = require('assert')
+
+const App = require('../../components/app/app')
```

Comment:
I think your case insensitive file system is showing. :D 1) This should be `../../components/App/App but 2) it could really be `../../components/App` because of the package.json in the App/ directory.